include/solvers/postprocessing_cfd.h

```diff
@@ -274,6 +273,62 @@ calculate_flow_rate(const DoFHandler<dim> &    dof_handler,
                     const Quadrature<dim - 1> &face_quadrature_formula,
                     const Mapping<dim> &       mapping);
 
+/**
+ * @brief calculate_average_velocity. This function calculates average velocity
+ * from the volumetric flow rate at the selected boundary.
+ *
+ * @param dof_handler. The argument used to get finite elements
+ *
+ * @param present_solution. The vector which contains all the values to
+ *                          calculate the flow rate
+ *
+ * @param boundary_id. The inlet boundary
+ *
+ * @param face_quadrature_formula The face quadrature formula for the calculation
+ *
+ * @param mapping The mapping of the simulation
+ */
+template <int dim, typename VectorType>
+double
+calculate_average_velocity(const DoFHandler<dim> &    dof_handler,
+                           const VectorType &         present_solution,
+                           const unsigned int &       boundary_id,
+                           const Quadrature<dim - 1> &face_quadrature_formula,
+                           const Mapping<dim> &       mapping);
+
+/**
+ * @brief calculate_average_velocity. This function calculates the average velocity of
+ * the domain for the solver GLS VANS.
+ *
+ * Ū = ∫ε(v⋅γ)dΩ/∫εdΩ
+ * Where Ū is the average velocity, ε is the void fraction, v is the velocity
+ * of the cell, γ is the vector of the cell (flow direction) and dΩ is the
+ * volume of the cell.
```

Comment:
I am not sure most text editor will like these characters. I would suggest using latex notation instead it's quite common.

Reply:
Oh ok then!